core/bentley/src/Compare.ts

```diff
@@ -92,3 +92,48 @@ export function areEqualPossiblyUndefined<T, U>(t: T | undefined, u: U | undefin
   else
     return areEqual(t, u);
 }
+
+/** @beta */
+export function compareSimpleTypes(lhs: number | string | boolean, rhs: number | string | boolean): number {
+  // Make sure the types are the same
+  if (typeof lhs !== typeof rhs) {
+    return 1;
```

Comment:
Violates strict-weak ordering, return the comparison of the strings instead to produce consistent ordering.

Reply:
fixed

Reply:
@pmconne only this comment is left unresolved